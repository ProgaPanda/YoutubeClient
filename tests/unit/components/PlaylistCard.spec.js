import { shallowMount } from '@vue/test-utils';
import PlaylistCard from '@/components/PlaylistCard.component.vue';

describe('PlaylistCard', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(PlaylistCard, { stubs: ['PlaylistIcon'] });
    expect(wrapper.element).toMatchSnapshot();
  });
});
