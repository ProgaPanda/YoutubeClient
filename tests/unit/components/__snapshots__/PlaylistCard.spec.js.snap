// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaylistCard renders correctly 1`] = `
<div
  class="playlist-card"
>
  <div
    class="playlist-card__thumbnail"
  >
    <img
      alt="video-thumbnail"
    />
     
    <playlisticon-stub
      class="playlist-card__thumbnail__icon"
    />
  </div>
   
  <div
    class="playlist-card__details"
  >
    <h4
      class="playlist-card__details__title"
    >
      
    </h4>
     
    <p
      class="playlist-card__details__meta"
    >
      
    </p>
     
    <p
      class="playlist-card__details__description"
    >
      
    </p>
  </div>
</div>
`;
